authentication/models.py

```diff
@@ -16,6 +16,8 @@ class Profile(models.Model):
         on_delete=models.CASCADE,
         related_name='profile'
     )
+    has_bank_linked = models.NullBooleanField(
+        blank=True, default=False, null=True)
```

Comment:
Seems unnecessary since this can be found with profile.user.bank == null

Reply:
Thought about that, but that query is a lot more expensive (has to search for ALL possible foreign keys - its not a one to one, so you wouldn't be able to check just for one). And seeing as its hitting home every time, I thought it would be more efficient to just have a boolean. I can change it though; let me know.